plugins/obs-webrtc/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+project(obs-webrtc)
+
+include(FetchContent)
+
+FetchContent_Declare(
+  Corrosion
+  GIT_REPOSITORY https://github.com/AndrewGaspar/corrosion.git
+  GIT_TAG origin/master # Optionally specify a version tag or branch here
+)
+
+FetchContent_MakeAvailable(Corrosion)
+
+include_directories(${LIBCURL_INCLUDE_DIRS})
+
+include_directories(${OBS_JANSSON_INCLUDE_DIRS})
+
+option(ENABLE_FFMPEG_LOGGING "Enables obs-ffmpeg logging" OFF)
```

Comment:
Old obs-ffmpeg cmake stuff. Can just nuke anything libav/ffmpeg related it appears.

Reply:
Fixed by 7a6c69765335b7ddb5adf0f94cab5baadff3bc58